README.md

```diff
@@ -106,6 +106,32 @@ const ComponentExample = () => {
 
 Pass `startWithTransition={true}` to automatically execute the `show` transition when the item initially mounts. It will be ignored if `show` is initially set to `false`.
 
+##### Additional performance consideration
+
+Since `Showtime` clones the child to attach its `ref`, it may be an expensive operation in some cases if the child component is substantially complicated. If so, provide a function that takes a `ref` and returns the child component instead, which will be more performant:
```

Comment:
A nit, but since we don't actually know for sure… "~will~ may be more performant"

Reply:
Added in 9a2ab8a